### PR TITLE
Change IntegrationRuns list method

### PR DESCRIPTION
**Pull request by mathialo**
API has been updated to follow the convention of other endpoints (ie, JSON is on the form `{ "items" : [ ... ] }` with pagination). Update SDK accordingly. 

Originally from an external PR (#152), but because of limited access to api-keys in the tests it has to go through an extra branch/PR.

### CODE REVIEW

File: cognite/experimental/data_classes/integrations.py [hunk 1]
```diff
@@ -80,6 +80,26 @@ class _PrimitiveIntegrationUpdate(CognitePrimitiveUpdate):
         def set(self, value: Any) -> "IntegrationUpdate":
             return self._set(value)
 
+    class _ObjectIntegrationUpdate(CogniteObjectUpdate):
+        def set(self, value: Dict) -> "IntegrationUpdate":
```
Comment (emelvarol-cognite) [inline cognite/experimental/data_classes/integrations.py:84]: I think I don't know this syntax. I thought -> shows the return type?

File: tests/tests_unit/test_api/test_runs.py [hunk 2]
```diff
@@ -18,17 +13,10 @@
 def mock_run_response(rsps):
```
Comment (emelvarol-cognite) [inline tests/tests_unit/test_api/test_runs.py:13]: Where do you use this?

**Review by emelvarol-cognite — APPROVED** [review]
Looks good

File: cognite/experimental/data_classes/integrations.py (hunk 1 above)
Comment (sanderland) [reply cognite/experimental/data_classes/integrations.py:84]: quoting a type hint is done for classes/types that are not yet defined.

**Pull request merged**